File: iCyption2022/practice/alex/solution.py
import math

theta, r, v, l = map(float, input().split())
	
alfa = theta / 2

jarakLingkaran = r / math.tan(math.radians(alfa))
jarakTongkat = l / math.tan(math.radians(theta))
	
jarakLingkaran = max(jarakLingkaran, jarakTongkat + r)

print("{:.2f}".format((100-jarakLingkaran)/v))

File: iCyption2022/practice/alex/solution.cpp
#include<bits/stdc++.h>
using namespace std;

typedef long double ld;

double r, l, v, theta, alfa, jarakLingkaran, jarakTongkat;

const ld PI = acos((long double) 0)*2;

int main (){
	
	cin >> theta >> r >> v >> l;
	
	alfa = theta / 2;
	
	jarakLingkaran = r / tan(alfa*PI/180);
	jarakTongkat = l / tan(theta*PI/180);
	
	jarakLingkaran = max(jarakLingkaran, jarakTongkat + r);
	
	cout << fixed << setprecision(2) << (100-jarakLingkaran)/v << endl;
	
}
